azure_billing/scrape.py

```diff
@@ -63,7 +63,7 @@ def extract_metrics_from_df(df, counter):
         counter.record(value.ExtendedCost, **meta)
 
 
-def data(enrollment, token, month=None):
+def data(enrollment, token, metric_name, month=None):
```

Comment:
done in https://github.com/blue-yonder/azure-cost-mon/pull/4/commits/d28b656d3d87885f11e134c39eca40e25ad8df20